### DISCUSSION TITLE
Welcome to StartOS Discussions!

### DISCUSSION THREAD

**Discussion by ProofOfKeags** [Start9Labs/start-os#384] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by k0gen** [comment]
Exercising my freedom in a new sovereign computing style that is emerging in this interesting times we all live in.

Be Happy

**Comment by alltheseas** [comment]
Embassy seems intriguing. Considering testing out when 0.3 is out. What are the hardware requirements?

**Reply by kn0wmad** [reply]
Check https://start9.com/eos-0.3.0 and https://docs.start9.com/getting-started/diy.html for hardware requirements, we will be re-writing these docs soon for more clarity.  We recommend an external hard drive rather than an internal with adapter.

**Comment by mpiraz** [comment]
I've just tried to install Embassy OS 0.3 by DIY approach but It didn't work I didn't hear two sounds when I connect power USB external disk and cable RJ45 . Router Vodafone station with my debian laptop and embassy connected at the same network. Anyone have some suggestions ? Thanks !

**Reply by kn0wmad** [reply]
Addressing via email.  For faster response, you can also find us on our community channels, listed at the bottom of start9.com
